src/onnx-mlir.cpp

```diff
@@ -44,9 +42,14 @@ int main(int argc, char *argv[]) {
     llvm::errs() << "Failed to parse options\n";
     return 1;
   }
-
   initCompilerConfig();
 
+  // timing manager reporting enabled via "--enable-timing" compiler flag
+  // mlir::applyDefaultTimingManagerCLOptions(timingManager);
+  timingManager.setEnabled(enableTiming);
+  rootScope = timingManager.getRootScope();
+  auto setupScope = rootScope.nest("[onnx-mlir] Preparing for Compilation");
```

Comment:
`setupScope` => `setupTiming`.

Perhaps, change the message to `[onnx-mlir] Loading Dialects`